LayoutTests/platform/mac-wk2/accessibility/aria-visible-element-roles-expected.txt

```diff
@@ -120,6 +120,10 @@ This test ensures ARIA visible elements (e.g. those with both `hidden` and `aria
     AXRole: AXList
     computedRoleString: list
 
+<menu hidden="" aria-hidden="false" id="menu-list"></menu>
+    AXRole: AXList
+    computedRoleString: list
+
```

Comment:
Probably we need that for other platforms. Let's see what EWS outputs...

Reply:
It seems you need to update the gtk and win expectations too. You can grab the results from the build bots.

Reply:
Ok. How do I do that?